tests/run.rs

```diff
@@ -193,6 +212,17 @@ fn integration_pyo3_mixed() {
     ));
 }
 
+#[test]
+fn integration_pyo3_mixed_include_exclude() {
+    handle_result(integration::test_integration(
+        "test-crates/pyo3-mixed-include-exclude",
+        None,
+        "integration-pyo3-mixed-include-exclude",
+        false,
+        None,
+    ));
+}
+
```

Comment:
```suggestion
```

I think we can just drop `develop`, `editable` and `integration` tests for this new test crate, we only want to test `include` and `exclude`.